packages/nest/collection.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "nx/nest",
+  "version": "0.1",
```

Comment:
Should we extend `@nestjs/schematics`?

Reply:
Let's do it separately. I'd like to restore the working setup first.